tabcmd/commands/user/user_command.py

```diff
@@ -19,6 +20,9 @@ class Column(IntEnum):
     EMAIL = 6
 
 
+logger = log(__name__, "DEBUG")
```

Comment:
Should this be a user-provided setting and not be hardcoded at the DEBUG level?

Reply:
Thanks, I deleted this, we don't need it here since the errors are indeed logged via error raising.